react-frontend/src/components/CaseStudies/horizontalinfocard.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import 'antd/dist/antd.css';
+import { Card } from 'antd';
+import '../../css/card.css';
+
+function HorizontalInfoCard(props) {
```

Comment:
I'll check what this looks like once deployed but you might be able to use InfoCard, I wasn't clear on what was possible with the component and conditional props but it might be able to work.